README.md

```diff
@@ -1,14 +1,21 @@
-# faq
-Community driven FAQ
+<p align="center">
+   <img width="150" src="https://raw.githubusercontent.com/floblockchain/flo-graphics/master/rebrand/FLO_black_nobg.png">
+</p>
 
-## Q. When was FLO created and launched?
-## A. Flow was created and launched in 2013. It is one of the oldest surviving cyrptocurrencities. Its Genesis Block was mined on    June 17, 2013.
+# FLO - FAQ
+Community driven FAQ for the FLO blockchain project - send us a pull request with questions or answers!
 
-## Q. I've heard FLO was established via a "fair launch." What does that mean?
-## A. A "fair launch" means there was no ICO, no premine, and no founders’ reward.
+### Q. When was FLO created and launched?
+_last update: May 31st 2019_  
+A. Flow was created and launched in 2013. It is one of the oldest surviving cyrptocurrencities. Its Genesis Block was mined on    June 17, 2013.
```

Comment:
Working from a mobile device? Seems autocorrect strikes again.

Reply:
Oops. Will fix

Reply:
I got it